demo/kubernetes/rook-client/rook-client.yml

```diff
@@ -2,17 +2,15 @@ apiVersion: v1
 kind: Pod
 metadata:
   name: rook-client
+  namespace: rook
 spec:
   containers:
   - name: rook-client
-    image: quay.io/rook/k8s-demo
+    image: quay.io/rook/rook-client
```

Comment:
i better not merge this rook-client.yml until we're ready to point the examples at the full k8s demo.